### PR TITLE
- fix fields  uncontrolled mode

### PR DESCRIPTION
**Pull request by singggum3b**
@mvtnghia  read this MR to reflect what go wrong. I had to fix a lot of thing...

### CODE REVIEW

File: field/select.field.js [hunk 1]
```diff
@@ -50,70 +50,89 @@ export default class SelectField extends React.Component {
...
 		return (
-			<label
-				className={cls}
-				key={itemProps.key}
-				onMouseEnter={this.onMouseEnter}
-				onMouseLeave={this.onMouseLeave}
-			>
-				<input type="checkbox" value={itemProps.key} onChange={this.onChange} className="checkbox" />
-				{itemProps.title}
+			<label htmlFor={this.props.key}
```
Comment (nghiamvt) [inline field/select.field.js:71]: if label is wrapper, it does not need htmlFor?

File: field/select.field.styl [hunk 2]
```diff
@@ -1,7 +1,11 @@
 @require "kouto-swiss"
 @require "global"
-.g-dropdown-list
-	.dropdown-item
+
+.g-field-select
+	&.g-field-select--multiple
+		.field-item
+			display: inline-block
+	.field-item
```
Comment (nghiamvt) [inline field/select.field.styl:8]: Have u tested it yet? 'cause I don't see any field-item inside of g-field-select

File: field/select.field.js (hunk 1 above)
Comment (singggum3b) [reply field/select.field.js:71]: yes. also htmlFor is trouble some since the input need to have unique id

File: field/select.field.styl (hunk 2 above)
Comment (singggum3b) [reply field/select.field.styl:8]: select field style file should not have g-dropdown as the namespace .
I've told you about this, that's each stylus file should only have 1 class as namespace.
Furthermore why are you styling a select field with a dropdown class ?

**Comment by singggum3b** [comment]
@mvtnghia I've done fixing overall for fields. Please read 2 commit :  first for uncontrolled mode, the other is for controlled form &  file field.
As a punishment for not testing thing thoroughly , pls clean up all the remaining eslint error 🚀

**Comment by singggum3b** [comment]
Approved

[Approved with PullApprove](https://pullapprove.com/singggum3b/react-components/pull-request/16/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=singggum3b/react-components)

**Pull request merged**